TFG-Labs/custom-shipping-step-by-items
# update furniture standard times

### What problem is this solving?
https://tfginfotec.atlassian.net/browse/BPF-1000?atlOrigin=eyJpIjoiN2NmOGRhMmI2MmI4NGQxZTljMDRlYmM4ZGM0MGRhNTYiLCJwIjoiaiJ9

#### How it works

#### How to test it?
- add furniture only items to cart -> delivery time should be 5 - 10 working days
- mixed furniture -> delivery should be 3 - 10 working days

[Workspace](https://bpf1000--thefoschini.myvtex.com/checkout)

### Screenshots/Video example usage:
- Desktop
- Tablet
- Movil

### Related to / Depends on

#### How does this PR make you feel? :link:
![]()
